src/coreclr/clr.featuredefines.props

```diff
@@ -76,4 +76,13 @@
         <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);TARGET_WINDOWS</DefineConstants>
         <DefineConstants Condition="'$(TargetsOSX)' == 'true'">$(DefineConstants);TARGET_OSX</DefineConstants>
     </PropertyGroup>
+
+  <!-- PYTHON will not be defined when building in VS-->
```

Comment:
Why is this the case? Can't see how VS would or would not interfere with this definition.

Reply:
This is outdated from a previous revision. I need to go remove this.